pages/02-extending/index.html

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
 title: USWDS Tutorial
-permalink: extend
+permalink: /extend.html
```

Comment:
This was flagged by the 11ty migration tool.

[More information →](https://www.11ty.dev/docs/permalinks/#trailing-slashes)